python312Packages.wallbox: 0.8.0 -> 0.9.0

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for python312Packages.wallbox is: Module for interacting with Wallbox EV charger API

meta.homepage for python312Packages.wallbox is: https://github.com/cliviu74/wallbox

meta.changelog for python312Packages.wallbox is: https://github.com/cliviu74/wallbox/releases/tag/0.9.0


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 0.9.0 with grep in /nix/store/vks66qxwz571l3dfzpkz5ylpm9iyaxmy-python3.12-wallbox-0.9.0
- found 0.9.0 in filename of file in /nix/store/vks66qxwz571l3dfzpkz5ylpm9iyaxmy-python3.12-wallbox-0.9.0

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
4 total rebuild path(s)

4 package rebuild(s)

First fifty rebuilds by attrpath

home-assistant-component-tests.wallbox
python312Packages.wallbox
python313Packages.wallbox
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/vks66qxwz571l3dfzpkz5ylpm9iyaxmy-python3.12-wallbox-0.9.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A python312Packages.wallbox https://github.com/r-ryantm/nixpkgs/archive/4f72f816b1bdd47c50dee859724f845b9234b815.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/4f72f816b1bdd47c50dee859724f845b9234b815#python312Packages.wallbox
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/vks66qxwz571l3dfzpkz5ylpm9iyaxmy-python3.12-wallbox-0.9.0
ls -la /nix/store/vks66qxwz571l3dfzpkz5ylpm9iyaxmy-python3.12-wallbox-0.9.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 5 packages built:</summary>
  <ul>
    <li>home-assistant-component-tests.wallbox</li>
    <li>python312Packages.wallbox</li>
    <li>python312Packages.wallbox.dist</li>
    <li>python313Packages.wallbox</li>
    <li>python313Packages.wallbox.dist</li>
  </ul>
</details>

---

###### Maintainer pings

cc @dotlambda for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

dotlambda reviewed (commented):
aenum is no longer a dependency

dotlambda committed:
python313Packages.wallbox: 0.8.0 -> 0.9.0

Diff: https://github.com/cliviu74/wallbox/compare/refs/tags/0.8.0...refs/tags/0.9.0

Changelog: https://github.com/cliviu74/wallbox/releases/tag/0.9.0
